README.md

```diff
@@ -97,6 +97,12 @@ end
     Z = promote_type(X,Y) # calculates Z from X and Y
     fun5(X,Y) -> Z
 end
+
+# using parametric trait. Note the nested curly
+@traitdef SemiFunctor{X{Y}} begin
+    fmap( Function, X{Y} } -> Any
+end
+
```

Comment:
Could the return type not be more specific here?  Otherwise there seems to be no need have a parametric type if none of the parameters actually feature.